From e534e871c7f22b3c0bb28d9f91f71b824cf68a0e Mon Sep 17 00:00:00 2001
From: "Francisco Blas (klondike) Izquierdo Riera" <klondike@klondike.es>
Date: Fri, 26 Feb 2021 11:15:08 +0100
Subject: [PATCH] Add missing format.h include in the upnp mapping management
 code to fix compile-time issues (Thanks juippis from Gentoo for finding and
 reporting this)

---
 dcpp/MappingManager.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/dcpp/MappingManager.cpp b/dcpp/MappingManager.cpp
index 42c4cbb4..c1d84f64 100644
--- a/dcpp/MappingManager.cpp
+++ b/dcpp/MappingManager.cpp
@@ -22,6 +22,7 @@
 #include "ConnectionManager.h"
 #include "SearchManager.h"
 #include "LogManager.h"
+#include "format.h"
 #include "version.h"
 #include "ConnectivityManager.h"
 #ifdef USE_MINIUPNP
-- 
2.26.2

